### ISSUE TITLE
[Bug]: Jet Suit does not check if on server or client.

### ISSUE THREAD

**Issue by PolyframeX** [terrarium-earth/Ad-Astra#212]
### Bug Description

When Changing The Client Side Config, Certain Changes Also Happen On The Server.

### How to Reproduce?

1. Use A Server
2. Get A Jet Suit
3. Change The Config Whilst In Game
4. See Changes Apply

### Expected Behavior

The Configuration Should Not Affect The Server. Such As Jet Suit Speed.

### Version

1.12.3

### Mod Loader Version

0.14.14

### Mod Loader

Fabric

### Logs or additional context

Nothing is outputted in logs related to the issue.

Video of issue is attached here:
https://drive.google.com/file/d/1QcD4L536B3WcuAlz1s_f4QRMb7rXmqoO/view?usp=sharing

**Comment by AlexNijjar** [comment]
is this actually affecting your movement or is it only affecting your client movement? if you have another player look at you on the server do they see you actually move faster?

**Comment by PolyframeX** [comment]
This actually affects my movement, so it's both a client/server issue.

**Comment by ThatGravyBoat** [comment]
Im gonna some info here, adastra_tick needs to check if on server only, current impl only sets free falling on server but not velocity causing this issue and likely causing an issue where if you set it high enough the server will kick you for flying hacking or moving too fast.

**Comment by PolyframeX** [comment]
I have it where allowing flying is enabled. So thats why I didn't get kicked probally earlier.

**Comment by PeaceIsBliss** [comment]
I can also confirm this. Realized it when I went to move my Energy bar away from my mini-map. Messed around with stuff and realized I could set my vertical flight to 0.1, and elytra flight to 1.0. Now I zip around the end till i spot Unobtanium, than practically hover in place while I mine it, and move along.

While this could easily be abused, I am actually appreciating it quite a lot, as it allows me control over situational environments, as makes the Jet Suit more valuable to me than say, elytra, creative flight, or Jet Packs.

But again, confirm, settings set on client override server defaults for speeds. I tested this as well with Oxygen limits and that however did not persist (or at least the existing Item did not like it very much, had weird results but the amount would not increase when attempting to fill with oxygen)
